ts/Series/Organization/OrganizationSeries.ts

```diff
@@ -410,6 +410,19 @@ class OrganizationSeries extends SankeySeries {
             shapeArgs.height;
     }
 
+    public drawDataLabels(): void {
+        const dlOptions = this.options.dataLabels;
+
+        if (dlOptions.linkTextPath && dlOptions.linkTextPath.enabled) {
+            for (const link of this.points) {
+                link.options.dataLabels = merge(link.options.dataLabels,
+                    { useHTML: false });
```

Comment:
This exception for `useHTML` - is that specific for org series? If not, we should refactor this so that it is handled for every series type that can use `textPath` for data labels, without repeating the code. It may be that it should be handled in the SVG renderer itself.

Reply:
We wanted to render the entire org series with `useHTML = true` by default, but if the user enables `linkTextPath` then we turn off the `useHTML` only for the links while the rest of the `dataLabels` are rendered with `useHTML = true`.

Do you think that whenever the `textPath.enabled = true` we should automatically disable `useHTML`? 

If yes, then maybe I'll create another issue for it & list all the names of the series which need to have it changed. This note: `Note: Only SVG-based renderer supports this option. Setting useHTML to true will disable this option.` should be removed from the docs for all of them.

Reply:
Okay, I understand what you are thinking. Org charts are special because we encourage `useHTML`. Let's just keep it as it is then.

Reply:
> Do you think that whenever the textPath.enabled = true we should automatically disable useHTML?

Can you follow that lead for a bit? If we can do that in a simple way, it would probably be the best way.

Reply:
> Do you think that whenever the textPath.enabled = true we should automatically disable useHTML?

I'm sorry I wasn't clear. I meant to ask if we should automatically disable `useHTML` if `textPath.enabled = true` for other series than organization chart. For example, for pie series, if `series.pie.dataLabels.textPath.enabled`, should we automatically set `series.pie.dataLabels.useHTML` to `false`? 

But you're right - in org charts we encourage `useHTML` and that's why I think it'll be fine if we do this exception for `useHTML` only here for now.